File: Language_Coder/src/loop/Code129.java
package loop;

import java.util.Scanner;

public class Code129 {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		while (true) {
			System.out.print("Base = ");
			double base = scan.nextDouble();

			System.out.print("Height = ");
			double height = scan.nextDouble();

			System.out.printf("%s = %.1f\n", "Triangle width", base * height * 0.5);

			System.out.print("Continue? ");
			char command = scan.next().charAt(0);

			if (command == 'Y' || command == 'y')
				continue;
			else
				break;
		}
	}

}

File: Language_Coder_Cpp/Code129/Code129.cpp
// Code129.cpp

#include <iostream>
using namespace std;

int main()
{
	while (true) {
		cout << "Base = ";
		int base; // Base
		cin >> base;

		cout << "Height = ";
		int height; // Height
		cin >> height;

		double area = base * (double)height * 0.5;

		cout << fixed;
		cout.precision(1);

		cout << "Triangle width = " << area << endl;

		cout << "Continue? ";

		string command;
		cin >> command;
		if (command == "Y" || command == "y") {
			continue;
		}
		else {
			break;
		}
	}
}